python_modules/dagster/dagster/core/definitions/composition.py

```diff
@@ -334,6 +334,21 @@ def __getitem__(self, idx):
             )
         )
 
+    def __getattr__(self, attr_called):
+        if attr_called == 'alias':
+            raise DagsterInvalidDefinitionError(
+                'In {source} {name}, attempted to call {attr_name} method for {cls}. This object '
+                'represents the output "{out}" from the already invoked solid "{solid}". Did you '
+                'forget to move parentheses?'.format(
```

Comment:
> Did you forget to move parentheses?

this statement reads a bit odd to me, I think a statement about how to remedy as opposed to a question assuming past intent is more clear,  "Double check the location of parentheses" or something like that

Reply:
Done.